docs/langref/relay_op.rst

```diff
@@ -51,8 +76,28 @@ Level 1 Definitions
 .. autofunction:: tvm.relay.sqrt
 .. autofunction:: tvm.relay.exp
 .. autofunction:: tvm.relay.add
+.. autofunction:: tvm.relay.subtract
+.. autofunction:: tvm.relay.multiply
+.. autofunction:: tvm.relay.divide
+.. autofunction:: tvm.relay.mod
+.. autofunction:: tvm.relay.pow
+.. autofunction:: tvm.relay.tanh
+.. autofunction:: tvm.relay.sigmoid
 
 
 Level 2 Definitions
 -------------------
 .. autofunction:: tvm.relay.nn.conv2d
+.. autofunction:: tvm.relay.nn.max_pool2d
+.. autofunction:: tvm.relay.nn.avg_pool2d
+.. autofunction:: tvm.relay.nn.global_max_pool2d
+.. autofunction:: tvm.relay.nn.global_avg_pool2d
+.. autofunction:: tvm.relay.upsampling
+
+Level 3 Definitions
+-------------------
+.. autofunction:: tvm.relay.batch_flatten
+
+Level 5 Definitions
+-------------------
+.. autofunction:: tvm.relay.vision.resize
```

Comment:
do we need to distinguish between image and vision? as resize is more a image operator

Reply:
Tricky call as we have very few ops in each category.

Reply:
In topi, resize belongs to topi.image.

Reply:
Can I take conclusion to go with new category image for resize?